src/rpc/rawtransaction.cpp

```diff
@@ -263,13 +300,33 @@ static RPCHelpMan getrawtransaction()
         throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, errmsg + ". Use gettransaction for wallet transactions.");
     }
 
-    if (!fVerbose) {
+    if (verbosity == 0) {
```

Comment:
Specifying a negative verbosity level fallbacks to 2 instead of 0.
```suggestion
    if (verbosity <= 0) {
```